pipeline/data_merging/brca_pseudonym_generator.py

```diff
@@ -85,6 +83,16 @@ def from_field_or_compute(row):
     return s_cdna
 
 
+def compute_genomic_hgvs(cDNA, assemblyMapper):
+    try:
+        genomic_hgvs = assemblyMapper.c_to_g(cDNA)
+        return str(genomic_hgvs)
+    except hgvs.exceptions.HGVSInvalidIntervalError as e:
+        logging.info("HGVSInvalidIntervalError: The given coordinate is outside the bounds of the reference sequence: " +
+                     str(cDNA))
```

Comment:
I'd catch the `HGVSError`, which is a base class for all sorts of HGVS issues. Also, I'd give some generic message along with the `cDNA` with the message from the actual error, like so:
```
logging.info("Exception during conversion of " + str(cDNA) + " to genomic coordinates: " + str(e)
```

Reply:
good call, this is now resolved :)